pyne/mesh.py

```diff
@@ -1395,8 +1395,10 @@ def tag_cell_fracs(self, cell_fracs):
             cell changing fastest.
 
         """
-
         num_vol_elements = len(self)
+        # sort cell_fracs
+        cell_fracs = _cell_fracs_sort_vol_frac_reverse(cell_fracs, num_vol_elements)
+        #cell_fracs.sort(order=['idx', 'vol_frac'])
```

Comment:
This commented line can probably go, yeah?

Reply:
Yes, removed this comment in 5abacf8.